fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -1813,25 +1862,41 @@ public OnlineIndexer build() {
             if (indexingPolicy == null) {
                 indexingPolicy = IndexingPolicy.DEFAULT;
             }
-            // Here: indexingPolicy is defined
-            Config conf = new Config(limit, maxRetries, recordsPerSecond, progressLogIntervalMillis, increaseLimitAfter, maxWriteLimitBytes);
-            return new OnlineIndexer(runner, recordStoreBuilder, index, recordTypes,
-                    configLoader, conf, syntheticIndex,
-                    useSynchronizedSession, leaseLengthMillis, trackProgress, indexingPolicy);
-        }
-
-        protected void validate() {
```

Comment:
I think the idea with having a single `validate()` method here was just to make it easier if we ever wanted to validate more things. Then you could add the method to this list, and the caller wouldn't need to be aware of what things were on that list. This is sort of moot if there's only one caller, but I think it makes sense as an abstraction.